src/app/screens/LNURLAuth/index.tsx

```diff
@@ -14,14 +14,14 @@ import { USER_REJECTED_ERROR } from "~/common/constants";
 import api from "~/common/lib/api";
 import msg from "~/common/lib/msg";
 import utils from "~/common/lib/utils";
-import type { LNURLAuthServiceResponse } from "~/types";
+import type { LNURLAuthServiceResponse, OriginData } from "~/types";
 
 function LNURLAuth() {
   const navigate = useNavigate();
 
   const navState = useNavigationState();
   const details = navState.args?.lnurlDetails as LNURLAuthServiceResponse;
-  const origin = navState.origin;
+  const origin = navState.origin as OriginData;
```

Comment:
I would add a small comment here `// origin is always defined for LNURLAuth`
For our future selves.